contracts/SnappAuction.sol

```diff
@@ -26,7 +26,20 @@ contract SnappAuction is SnappBase {
     mapping (uint16 => StandingOrderData) public standingOrders;
 
     uint public auctionIndex = MAX_UINT;
-    mapping (uint => SnappBaseCore.PendingBatch) public auctions;
+
+    struct PendingAuction {
+        address solver;
+        uint objectiveValue;           // Traders utility
+        bytes32 solutionHash;          // Succinct record of trade execution & prices
+        uint solutionAcceptedTime;   // Time solution was accepted (written at time of solutionHash)
+        bytes32 tentativeState;        // Proposed account state during bidding phase
+        uint16 size;                   // Number of orders in this auction
```

Comment:
ubernit: if small variables are at all together (at then end of the struct), solidity can collate them into a single word.

Maybe we can also call this something like number of orders?

Reply:
I don't understand your ubernit.

I wont change the `size` here, because I will like use the `PendingBatch` struct from the `SnappBase`

Reply:
Why do we need to store the size of orders?
I think in all fraud proofs, we can deduct the size from the solution submission.

Reply:
The size is needed in order to emit the order index (i.e. so the contract knows which order it is).